active/AnimatedIcon/AnimatedIconSpec.md

```diff
@@ -0,0 +1,894 @@
+```
+
+# API Pages
+
+_(Each of the following L2 sections correspond to a page that will be on docs.microsoft.com)_
+
+## AnimatedIcon class
+
+An icon that displays and controls an IAnimatedVisual2, which is set as the value of the Source property.
+
+You can define an IAnimatedVisual2 using the
+[Lottie-Windows](https://docs.microsoft.com/en-us/windows/communitytoolkit/animations/lottie) library.
+
+You change the position of the animation displayed by an AnimatedIcon by setting its `State` attached property
+on the AnimatedIcon or on its parent in the XAML tree.
+(If you add an AnimatedIcon to the tree and the property is already set on the new parent, the icon will move to that state.)
+
+Using this mechanism you can easily add an AnimatedIcon to a XAML control's ControlTemplate and set its state using
+[VisualStateManager](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.VisualStateManager).
+Several XAML controls have this support already built in.
+
+### How is AnimatedIcon different than [AnimatedVisualPlayer](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer)?
+
+AnimatedIcon is an
+[IconElement](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.IconElement),
+and so can be used anywhere an element or IconElement is required, such as
+[NavigationViewItem.Icon](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.NavigationViewItem.Icon).
+`AnimatedVisualPlayer` is a more  general animation player that can be used anywhere in an application where AnimatedIcon is intended to
+only be used where icons are used in WinUI.
+
+### Example
+
+The following example is a button that the user clicks to Accept a prompt.
+The MyAcceptAnimation is a class created using the LottieGen tool.
+The FallbackIconSource will be used rather than the animation when animations can't be played,
+such as on older versions of Windows that don't support Lottie animations.
+
+```xml
+<Button PointerEntered="HandlePointerEntered" PointerExited="HandlePointerExited">
+    <StackPanel Orientation='Horizontal' AnimatedIcon.State='Normal'>
+
+        <AnimatedIcon x:Name='AnimatedIcon1'>
+            <local:MyAcceptAnimation/>
+            <AnimatedIcon.FallbackIconSource>
+                <SymbolIconSource Symbol='Accept'/>
+            </AnimatedIcon.FallbackIconSource>
+        </AnimatedIcon>
+
+        <TextBlock>Accept</TextBlock>
+    </StackPanel>
+</Button>
+```
+
+```cs
+private void Button_PointerEntered(object sender, PointerRoutedEventArgs e)
+{
+    AnimatedIcon.SetState(this.AnimatedIcon1, "PointerOver");
+}
+
+private void Button_PointerExited(object sender, PointerRoutedEventArgs e)
+{
+    AnimatedIcon.SetState(this.StackPaAnimatedIcon1nel1, "Normal");
+}
+```
+
+## AnimatedIcon.State attached property
+
+```c#
+static public void SetState(UIElement element, string value);
+static public string GetState(UIElement element);
+```
```

Comment:
What's the default State?

Reply:
I believe it's "Normal" but I will let @StephenLPeters confirm.